### ISSUE TITLE
Instagram thumbnails request

### ISSUE THREAD

**Issue by modelfe** [qsniyg/maxurl#354]
Would it be possible for this extension to fetch Instagram profile pictures in full size? As, for example, the izuum.com website does?

Additionally, is there any way to display Instagram Story Highlights thumbnails in full resolution? I haven't been able to do this at all.

Cheers

**Comment by qsniyg** [comment]
Profile thumbnails should work in theory... but if not, could you link a profile that doesn't work?

Yeah that one has kind of been on my radar for a while, but I'm not really sure how to best approach it... it's one of those "try to vaguely guess that this is a story highlight from how it looks like on the page" things, thanks to obfuscated CSS classes haha.

**Comment by SirSaarim** [comment]
For some accounts it works and for some it doesn't. For example the following account it fails to allow right click and do its trick.

Link: https://www.instagram.com/realdonaldtrump/

**Comment by qsniyg** [comment]
@SirSaarim Right, that's because Instagram prevents you from right clicking on the images sometimes. Under Firefox, you can bypass this by pressing shift+right click, but I don't think the same exists for Chrome unfortunately.

However, you can also invoke the popup through a trigger, either keyboard (set to shift+alt+i by default), or you can set it to open when you hover over an image. This should allow you to open the popup regardless of whether right click is disabled or not :)

**Comment by Jon8RFC** [comment]
This works and has for a while.  This can be closed.

Found this while trying to search for another thumbnail issue.
